repo/packages/C/crate/0/marathon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "/crate/{{crate.cluster-name}}",
```

Comment:
the id is the marathon id, it would be nice to have the marathon id closely resemble the framework name... 

please use the framework-name for the id.